locale/action_names.rb

```diff
@@ -1,33 +1,36 @@
 # Autogenerated!
 _("Abstract async task")
 _("Abstract")
-_("Action with sub plans")
 _("Attach subscriptions")
 _("Auto attach subscriptions")
 _("Bulk generate applicability for hosts")
-_("Check for long running tasks")
 _("Combined Profile Update")
 _("Copy version units to library")
 _("Create Alternate Content Source")
 _("Create Export History")
 _("Create Import History")
 _("Create Syncable Export History")
 _("Create")
+_("Create")
+_("Create")
```

Comment:
@ianballou any idea what's going on with these duplicates?

Reply:
When re-running the translation pulls from Transifex, we encountered the exact same duplicate strings in the exact same locations. See https://github.com/Katello/katello/pull/10877

Reply:
This led William and I to believe these duplicates are somehow intentional. We're going to resolve these issues.